public/js/interface.js

```diff
@@ -0,0 +1,116 @@
+var calculateTotal = function(){
+  $.get("/total", function(data) {
+    $("#totalPrice").text("Total: £" + data.total);
+    $("#tax").text("Tax (8.64%): £" + data.tax);
+    $("#afterTax").text("After Tax: £" + data.after);
+  });
+};
+
+var addToList = function(){
+  $.get("/items", function(data) {
+     calculateTotal();
+     $("#errorTill").empty();  
+     $("#tillNumbers").empty();
+    var size = +data.item.length;
+    for (var i = 0; i < size; i++) {
```

Comment:
There's nicer ways of looping through the code rather than `for` loops they're a bit inelegant, for example jquery has an `each` method